test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3109,12 +3109,48 @@ def setup_rnn_tests():
 
 setup_rnn_tests()
 
+# Unary floating ufunc tests
+# Tests that integral arguments are properly cast to float
+unary_floating_ufuncs = (
+    'sin', 'cos', 'tan',
+    'asin', 'acos', 'atan',
+    'sinh', 'cosh', 'tanh',
+    'ceil', 'floor',
+    'exp', 'log', 'log1p', 'log2',
+    'sqrt', 'rsqrt',
+    'erf',
+    'sigmoid'
+)
+
+def setup_unary_floating_ufunc_tests_helper(cls, op_name):
+    op = getattr(torch, op_name)
+
+    def test_fn(self):
+        class module(torch.nn.Module):
+            def forward(self, x):
+                return op(x)
+
+        x = (torch.rand(2, 3, 4) * 10).to(torch.long)
+        self.run_test(module(), (x,))
+        self.run_test(torch.jit.trace(module(), (x,)), (x,))
+
+    test_name = "test_" + op_name + "_ufunc"
+    assert not hasattr(cls, test_name), "{0} already in {1}".format(test_name, cls.__name__)
+    if(op_name in ['erf', 'sinh', 'cosh']):
```

Comment:
`if op_name in [...]:`

would be more pythonic

Reply:
Agreed! This is solved in the recent commit: https://github.com/pytorch/pytorch/pull/33322/commits/29ae207f362efe4040b598355cfe1db1fadb82ad. Thanks!